### DISCUSSION TITLE
Discussing tsinfer and tsdate

### DISCUSSION THREAD

**Discussion by lakishadavid** [tskit-dev/tsinfer#543] [Q&A] [answered]
Searching for tsinfer on StackOverflow does not produce results. Is there a place to discuss usage of tsinfer and tsdate more generally? The questions I have are more basic, and I not sure if I should clutter the discussion space here with my several of those types of questions.

**Comment by benjeffery** [comment, marked as answer]
That's exactly what this discussion place is for, please go ahead!

**Comment by arpfennig** [comment]
Hi,

Thanks for developing tsinfer and tsdate. I just started using it, but I'm impressed by its capabilities and computational efficiency. I got a question on how to prepare my input data. 

I'm attempting to compare the age of mutations on two classes of haplotypes. Both classes have similar length distributions (~50 - 1,000 kb; median: ~80kb). I'm currently running it for only a few hundred 1KGP individuals but eventually want to scale it up to ~20k individuals. For the sake of computational costs, I was wondering if I could run tsinfer and tsdate on individual haplotypes rather than whole chromosomes? Or would the data be too sparse to obtain robust results?

Thanks for your suggestions,
Aaron

**Reply by jeromekelleher** [reply]
Hi @AaronRuben :wave:

We tend to run tsinfer on chromosome arms these days @AaronRuben, as there's unlikely to be much information across centromeres and it does help reduce the computational cost quite a bit. We often use small chunks of a chromosome when doing development work, as you can get meaninful trees on (say) 10% of a chromosome arm. How much is "enough" is something that varies quite a lot, and I don't think there's a general answer to this.

Just for future reference we'd usually start a new discussion [here](https://github.com/tskit-dev/tsinfer/discussions) for questions like this.
